### DISCUSSION TITLE
Notepad demo no such table error

### DISCUSSION THREAD

**Discussion by JasperAlexander** [payton/django-siwe-auth#49] [Q&A] [answered]
First of all, thank you for creating this repository!

When I click on Sign-In with Ethereum at the demo app (Notepad), I'm getting the error: `django.db.utils.OperationalError: no such table: app_notepad` caused by the line `notepad, created = Notepad.objects.get_or_create(pk=request.user.ethereum_address)` at app/views.py. How to solve this?

I've followed all the steps described in the README file. Something strange is that I don't see migrations folders after running `poetry run examples/notepad/manage.py makemigrations` and `poetry run examples/notepad/manage.py migrate`.

**Comment by payton** [comment, marked as answer]
Hey @JasperAlexander! You're totally right. Thanks for bringing this up.

My sqlite file had this model already applied from a while ago, so I never ran into the issue. Looks like you just need to run `makemigrations` on the app with the Notepad/SharedNotepad models:

`poetry run examples/notepad/manage.py makemigrations app`

Want to make a PR to update instructions for this based on your experience?

**Reply by JasperAlexander** [reply]
Thank you! I will do it tomorrow.
